### PR TITLE
#63 upgraded to kiama 2.1.0-RC1 and scala 2.12.2

### PR DESCRIPTION
**Pull request by etorreborre**
Review @jcranky

### CODE REVIEW

**Comment by yvoderatskyi** [comment]
👍

**Comment by etorreborre** [comment]
👍

File: core/src/main/scala/org/zalando/grafter/Queries.scala [hunk 1]
```diff
@@ -47,22 +47,22 @@ trait Query {
...
-  type Path = List[Any]
+  type Path = List[Product]
 
   /**
    * collect ancestor paths of a node of type T in a graph and show simple class names
    *  a list of pairs is returned where there can be duplicates in the key because
    *  there can be several instances of the same type
    */
-  def ancestorNames[T : ClassTag, G <: Product](graph: G): List[(String, List[List[String]])] =
+  def ancestorNames[T <: Product : ClassTag, G <: Product](graph: G): List[(String, List[List[String]])] =
```
Comment (jcranky) [inline core/src/main/scala/org/zalando/grafter/Queries.scala]: A bit more specific, nice

**Comment by jcranky** [comment]
:+1:

**Comment by etorreborre** [comment]
👍

**Comment by etorreborre** [comment]
👍

**Comment by alexkops** [comment]
:+1:

**Pull request merged**